src/sage/arith/misc.py

```diff
@@ -82,109 +82,109 @@ def algdep(z, degree, known_bits=None, use_bits=None, known_digits=None,
 
     EXAMPLES::
 
-        sage: algdep(1.888888888888888, 1)
+        sage: algdep(1.888888888888888, 1)                                                                              # optional - sage.libs.pari
```

Comment:
I don't understand the alignment of the `# optional - ...` in this file. It varies between blocks. Furthermore, some lines have 2 `# optional - ...` tags.

Reply:
Indeed, this was completely messed up. Fixed now.